<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maintree</class>
 <widget class="QMainWindow" name="maintree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>maintree</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QPushButton" name="SearchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QComboBox" name="ScaleBox">
      <item>
       <property name="text">
        <string>5%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>75%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>125%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>150%</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="9">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="SearchValue"/>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="DeleteValue"/>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="DeleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="printBtn">
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QPushButton" name="ClearBtn">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="AddButton">
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="Value"/>
    </item>
    <item row="2" column="8">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Max Degree</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExample"/>
    <addaction name="actionSave_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExample">
   <property name="text">
    <string>Random Examples</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
